ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -17,6 +18,13 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
   @service session;
   @service store;
 
+  // =attributes
+
+  @tracked targets = this.model;
+  @tracked freeSearch;
+  @tracked searchItems = ['Project 1', 'Project 2', 'Project 3'];
```

Comment:
What are these `searchItems` for?  Is this for testing purposes?

Reply:
Yes, it was for testing purposes originally and what I will be working on next. I had converted that dropdown to be more like what it will look like in it's final iteration which will be a dynamic list of projects available for filtering. I decided to leave it in for now instead of hardcoding it in the template.

Reply:
not blocking comment: Could we add a short comment saying this is temporary and we will rely on real data later?